wordcloud/wordcloud.py

```diff
@@ -0,0 +1,46 @@
+import os
+from operator import itemgetter
+import re
+
+from haystack.query import SearchQuerySet
+from pombola.hansard import models as hansard_models
+
+
+BASEDIR = os.path.dirname(__file__)
+# normal english stop words and hansard-centric words to ignore
+STOP_WORDS = open(os.path.join(BASEDIR, 'stopwords.txt'), 'rU').read().splitlines()
```

Comment:
I think STOP_WORDS should be a set - since the code's only ever checking membership it'll be way faster. Also, I'd write this as:

```python
with open(os.path.join(BASEDIR, 'stopwords.txt'), 'rU') as f:
    STOP_WORDS = set(f.read().splitlines())
```

... for the nice file-closing-on-exceptions behaviour.